src/quacc/calculators/mrcc/mrcc.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import annotations
+
+import re
+
+from ase.calculators.genericfileio import (
+    BaseProfile,
+    CalculatorTemplate,
+    GenericFileIOCalculator,
+)
+
+import quacc.calculators.mrcc.io as io
```

Comment:
Since ASE also has an `io` module, I would recommend not doing this and instead do

```python
from quacc.calculators import mrcc
```

and then use `mrcc.io()`